lib/Conversion/TorchToLinalg/Reduction.cpp

```diff
@@ -152,12 +152,10 @@ class ConvertAtenMaxDimOp : public OpConversionPattern<AtenMaxDimOp> {
               nestedLoc, oldIndex.getType(),
               rewriter.create<linalg::IndexOp>(loc, dim));
 
-          Value predicate;
-          if (inElementType.isa<mlir::FloatType>())
```

Comment:
This `if` is not needed since it is already checked in the beginning of the function:

https://github.com/llvm/torch-mlir/blob/c14ce36f80b536e4306f5d4a5b71177d2d20c53c/lib/Conversion/TorchToLinalg/Reduction.cpp#L83